packages/clarity-js/src/data/metadata.ts

```diff
@@ -132,13 +165,22 @@ export function consent(status: boolean = true): void {
   }
 
   if (core.active()) {
+    trackConsent.consentv2(consentData);
     config.track = true;
     track(user(), BooleanFlag.True);
     save();
-    trackConsent.consent();
   }
 }
 
+function getConsent(status: Status): ConsentData {
+  let consent: ConsentData = {
+    ad_Storage: status.ad_Storage === Constant.Granted ? BooleanFlag.True : BooleanFlag.False,
```

Comment:
will consentStatus be case sensitive? what happens when you pass in Granted instead of granted